geoportal/tests/functional/__init__.py

```diff
@@ -170,21 +177,21 @@ def teardown_common() -> None:
     models.DBSessions = {}
 
 
-def create_default_ogcserver():
-    from c2cgeoportal_commons.models import DBSession
+def create_default_ogcserver(session: Session) -> "main.OGCServer":
     from c2cgeoportal_commons.models.main import OGCServer
 
-    transaction.commit()
-    ogcserver = OGCServer(name="__test_ogc_server")
+    session.flush()
```

Comment:
We should flush at end of changes instead of before.
It would be really easier to debug.
Error would raise in function which made a mistake.

Also note that here pretty sure next SELECT will raise a flush.

Reply:
Applied in #11460